modules/bot_assistant/utils/note_state.py

```diff
@@ -0,0 +1,20 @@
+from enum import Enum
+
+
+class State(Enum):
```

Comment:
Move class definition to models please if possbile

Reply:
done